src/Components/Buttons/index.js

```diff
@@ -0,0 +1,116 @@
+import React, {Component} from 'react'
+import './index.css' 
+
+export class Single extends Component {
+
+    state = {
+        activeButton: ''
+    }
+
+    toggleClick = (title) => {
+        this.setState({
+            activeButton: title
+        })
+    }
+
+    renderButtons = ({items, active}) => { 
```

Comment:
if we will use it like I wrote below: `{this.renderButtons(items, active)}`
so we should change renderButtons in such way: `renderButtons = (items, active) => { `(without curly braces)

Reply:
Can't get how is this working